src/main/resources/swagger/api-docs.yaml

```diff
@@ -1512,6 +1514,9 @@ definitions:
         type: array
         items:
           type: string
+      returnResource:
+        type: boolean
+        description: return created resource information with 201 response code
```

Comment:
add `required` and `default` keys, assuming this is optional

Reply:
see above for required, added `default`